wdl/munge.wdl

```diff
@@ -327,7 +326,6 @@ task lift_postprocess {
         s_f = gzip.open(sumstat, 'rt')
         sumstat_header = s_f.readline().strip().split(delim)
         sumstat_h_idx = {h:i for i,h in enumerate(sumstat_header)}
-        sumstat_h_idx = defaultdict(lambda: None, sumstat_h_idx)
```

Comment:
same here that better explicitly check compulsory columns and error out instead of just key error.